extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/GrpcUtil.java

```diff
@@ -137,7 +137,7 @@ public static void safelyExecute(final FunctionalInterfaces.ThrowingRunnable<Exc
         try {
             runner.run();
         } catch (final Exception err) {
-            log.debug().append("Unanticipated gRPC Error: ").append(err).endl();
+            log.error().append("Unanticipated gRPC Error: ").append(err).endl();
```

Comment:
What were we trying to suppress when it was `debug`?

Reply:
I'm not sure. I don't think @nbauernfeind remembers either.

Reply:
I'm going to suggest logging these as error, and re-evaluate later if we see too much logspam.